app/src/lib/stores/pull-request-store.ts

```diff
@@ -251,146 +308,105 @@ export class PullRequestStore extends TypedBaseStore<GitHubRepository> {
     )
   }
 
-  private async writePRs(
-    pullRequests: ReadonlyArray<IAPIPullRequest>,
+  private async cachePullRequests(
+    apiuPullRequestsFrom: ReadonlyArray<IAPIPullRequest>,
```

Comment:
- `apiu` -> `api`
 - does `From` need to be `FromServer` or `FromAPI` for some additional context?
 - actually why not `pullRequestsFromAPI`?